client/app/scripts/superdesk-packaging/packaging.js

```diff
@@ -608,7 +629,7 @@
                             scope.action = 'view';
                         });
                     }
-                });
+                }, true);
```

Comment:
not sure why you need this recursive watch, but it won't be available soon..
check #1060